pcsx2/gui/MainFrame.cpp

```diff
@@ -584,19 +587,66 @@ void MainEmuFrame::OnActivate( wxActivateEvent& evt )
 }
 // ----------------------------------------------------------------------------
 
+static void GetMenuItemLabel(MenuIdentifiers Id, wxString* label)
+{
+	const CDVD_SourceType Source = g_Conf->CdvdSource;
+	const bool ActiveVM = SysHasValidState();
```

Comment:
``ActiveVM`` is no longer needed as we removed the ``reboot`` option. Just a left over from my older code where the function was also initializing strings for the reboot menu item labels.